aws_lambda_powertools/utilities/idempotency/persistence/base.py

```diff
@@ -344,6 +350,14 @@ def save_inprogress(self, data: Dict[str, Any]) -> None:
             payload_hash=self._get_hashed_payload(data=data),
         )
 
+        if remaining_time_in_millis:
+            now = datetime.datetime.now()
+            period = datetime.timedelta(milliseconds=remaining_time_in_millis)
+
+            data_record.in_progress_expiry_timestamp = int((now + period).timestamp() * 1000)
```

Comment:
maybe split into a separate variable `(now + period).timestamp` so it's easier to read that you're using the adding the delta and multiplying by 1000 for ms